### DISCUSSION TITLE
list of 12 open ideas to make opencollective even better

### DISCUSSION THREAD

**Discussion by ghost** [opencollective/opencollective#6209] [Ideas]
Hi everyone.

I would like to help opencollective with some ideas. I have a list of 12 things or a list of 12 open ideas to make opencollective even better:
1. Open badge in OpenCollective: As the open source project gains more users, customers, developers, partners. This project must or can win an open badge, which is recognition of its activity, an "open achievement"(this reward is something symbolic and has no capitalist purpose).
2. All money can be returned to the community if the open source project goes down. Closed products offer 30 days of use for the user to get to know the product and, if he wants to, become a member or customer or end user. We could have this same idea if the open source project has no more users or the cost is too high to keep it running. There are consumer laws, which you can change products or services if you don't like a product or service for 7 days. If the community shuts down its activities, we can reallocate the money from that closed open source project to the same new open source project. This ensures greater transparency and control within the open source community.
3. Every open source project has to donate 10% to any social project. It increases the control, democratic participation of open source project.
4. Who can vote on features must be registered users of the open collective or developers within the open collective.
5. It is necessary that the open project demonstrates the open business model as: "open core", "free", "premium" etc in the open collective through the canva diagram that is used to develop, maintain and manage business models
6. I would like the interface of each open web project to be componentized by web components, block protocol, etc.
7. Every open project should create financial reports with open data for the community to check how community investments are coming and going
8. Every open project should be done on servers, open source companies like "Canonical", "RedHat", "FossHost" etc
9. Every open source project must have a website or page to clarify doubts, FAQ etc.
10. Open source projects should prioritize decentralized, distributed protocols like IPFS, ActivityPub, Hypercore, Dat, etc.
11. All open source projects must work in a way to be "self-hosted", "unhosted". That is, they must work without a backend or if there is a need for a backend, they must be self-hosted
12. There are open projects that only the client code is available, it is necessary to have the server and client code as well or any code that is necessary or important for the application to work

*What do you think of this idea?*

**Comment by aerinsol** [comment]
I like this, however, we would want the ability to vote to not necessarily be for developers. Most of the target audience for our project are humanitarian volunteers and coordinators and we would want their input regardless.
